app/_locales/en/messages.json

```diff
@@ -1722,14 +1722,14 @@
     "message": "Your initial transaction was confirmed by the network. Click OK to go back."
   },
   "inputLogicEmptyState": {
-    "message": "Only enter a number that you're comfortable with the contract spending now or in the future. You can always increase the spending cap later."
+    "message": "Only enter a number that you're comfortable with the third party spending now or in the future. You can always increase the spending cap later."
   },
   "inputLogicEqualOrSmallerNumber": {
-    "message": "This allows the contract to spend $1 from your current balance.",
+    "message": "This allows the third party to spend $1 from your current balance.",
```

Comment:
Add hypen
```suggestion
    "message": "This allows the third-party to spend $1 from your current balance.",
```

Reply:
in this case, the red text is right. no hyphen needed here!

Reply:
Ah my bad! Thanks @coreyjanssen